vsts-fullbuild-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vstsAccountURL": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services account URL"
+      }
+    },
+    "vstsPAT": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services PAT for user with build permissions"
+      }
+    },
+    "vstsPoolName": {
+      "type": "string",
+      "defaultValue": "default",
+      "metadata": {
+        "description": "Team Services Agent Pool Name"
+      }
+    },
+    "vstsAgentName": {
+      "type": "string",
+      "defaultValue": "fullbuild-ubuntu-auto",
+      "metadata": {
+        "description": "Team Services Agent Name"
+      }
+    },    
+    "newStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Storage Account"
+      }
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "imagePublisher": {
```

Comment:
Image properties should be a variable instead since the scenario depends on these and a different user input will probably break this template